frontend/javascripts/maintenance_banner.tsx

```diff
@@ -144,3 +148,83 @@ export function MaintenanceBanner() {
 
   return null;
 }
+
+export function UpgradeVersionBanner() {
+  const white = "var(--ant-color-text-primary)";
+  const blue = "var(--ant-color-primary)";
+  const UPGRADE_BANNER_STYLE: React.CSSProperties = {
+    position: "absolute",
+    top: 0,
+    left: 0,
+    height: constants.BANNER_HEIGHT,
+    textAlign: "center",
+    backgroundColor: blue,
+    color: white,
+    fontSize: "medium",
+    minWidth: "fit-content",
+  };
+  const customParseFormat = require("dayjs/plugin/customParseFormat");
+  dayjs.extend(customParseFormat);
```

Comment:
can you {move that code to} and {avoid the require call like it's done} [here](https://github.com/scalableminds/webknossos/blob/2eed0758dbcc0a8f00762c9f3a251a8c6286418d/frontend/javascripts/libs/format_utils.ts#L19)?